cdap-api/src/main/java/co/cask/cdap/api/service/http/HttpServiceHandler.java

```diff
@@ -56,7 +60,7 @@
  *
  * @see HttpContentConsumer
  */
-public interface HttpServiceHandler {
+public interface HttpServiceHandler extends ProgramLifecycle<HttpServiceContext> {
```

Comment:
Might be good to update the javadoc example to include usage of `@TransactionPolicy` on handler method (e.g. `process` in the above example) and to state clearly that the default is implicit.